skrifa/src/outline/mod.rs

```diff
@@ -0,0 +1,448 @@
+//! Loading, scaling and hinting of glyph outlines.
+
+mod cff;
+mod error;
+mod glyf;
+mod native_hint;
+
+use read_fonts::{types::GlyphId, TableProvider};
+
+pub use error::DrawError;
+pub use native_hint::{NativeHinting, NativeHintingInstance};
+
+pub use read_fonts::types::Pen as OutlinePen;
+
+use super::{
+    instance::{LocationRef, NormalizedCoord, Size},
+    GLYF_COMPOSITE_RECURSION_LIMIT,
+};
+
+/// Source format for an outline glyph.
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum OutlineGlyphFormat {
+    /// TrueType outlines sourced from the `glyf` table.
+    Glyf,
+    /// PostScript outlines sourced from the `CFF` table.
+    Cff,
+    /// PostScript outlines sourced from the `CFF2` table.
+    Cff2,
+}
+
+/// Specifies the hinting strategy.
+#[derive(Copy, Clone, PartialEq, Eq, Default, Debug)]
+pub enum Hinting {
+    /// Hinting is disabled.
+    #[default]
+    None,
+    /// Will apply native embedded hints.
```

Comment:
Suggest a wee bit more information to support the reader who is going wtf is a native hint :)

Reply:
(and what on earth is a non-native hint)

Reply:
I chose Native as the analogue to Auto for when auto hinting is implemented. Maybe Embedded is a better prefix?

Reply:
Went with the `Embedded` prefix and added more explanation. Happy to bikeshed and rename later if we come up with something better.